src/current/_includes/v22.2/backups/storage-collision-examples.md

```diff
@@ -0,0 +1 @@
+For example, if you have a backup schedule running backups for the database `users` the full backup and incremental backup should have the same storage URI for the full and incremental schedule. (`CREATE SCHEDULE FOR BACKUP` will automatically create two schedules for the full and incremental backup to the same storage URI.) If there is another backup schedule, for the database `accounts`, the full and incremental backups for `accounts` should have the same storage URI. However, the storage URI for the `accounts` backup collection, should be different to the storage URI for the `users` database backups.
```

Comment:
I think this would be slightly easier to understand if the final sentence used the same term for the things being contrasted eg something like 

> the storage URI for the A backup collection should be different from the storage URI for the B backup collection

Also no comma is needed after 'collection' in that sentence

Reply:
Oh yes! Very good point. I have adapted. Yep, not sure why I added a comma there! Thanks for catching 👍